### ISSUE TITLE
Task [db:drop] is running in [default_env] environment

### ISSUE THREAD

**Issue by FranklinYu** [jhollinger/otr-activerecord#10]
I have my `Rakefile` like this:

```ruby
load 'tasks/otr-activerecord.rake'
OTR::ActiveRecord.configure_from_file!(File.join(__dir__, 'config/database.yml'))
```

When I run `rake db:setup` the database is set up correctly. However, when I do `rake db:drop` it says

```text
ActiveRecord::EnvironmentMismatchError: You are attempting to modify a database that was last run in `development` environment.
You are running in `default_env` environment.
```

Issue goes away when I add `ENV['RACK_ENV'] ||= 'development'` in `Rakefile`, but is this the expected behavior? I thought `db:drop` should behave similar to `db:create`?

**Comment by jhollinger** [comment]
I've seen that error a couple times in an AR 5 project but haven't had time to investigate. Sounds like you found the answer.

Feel free to make a PR to default the env to that if you want. Otherwise I'll try to get out a patch later this week.

**Comment by jhollinger** [comment]
Fixed in 1.2.6. It will now properly default to `development`.

For existing projects you may need to run `db:migrate` (even if there are no migrations) or some other `db` command to totally remove `default_env` from wherever AR is keeping track of it.
